.papr.yml

```diff
@@ -19,9 +30,10 @@ packages:
   - openssl-devel
   - redhat-rpm-config
 
-context: 'fedora/25/atomic | origin/v3.6.0-alpha.1'
+context: 'fedora/25/atomic | origin/v3.6.0-alpha.2'
```

Comment:
Yeah, it's not very intuitive I admit. It refers to the name of the green checkmark in the GitHub interface. (I recently added a blurb in the [README](https://github.com/projectatomic/papr#papr-previously-called-redhat-ci) to clarify this.)

There's no way to make these hold variables for now, though it's an interesting suggestion. I filed https://github.com/projectatomic/papr/issues/42.